cellprofiler/modules/measurecolocalization.py

```diff
@@ -23,6 +23,8 @@
 -  The blue and green, red and green, and red and blue images.
 -  The nuclei in each of the above image pairs.
 
+A good primer on colocalization theory can be found on the `SVI website`_.
+
```

Comment:
Don't we have a colocalization white paper/tutorial on the cp.org website too (or is that already mentioned in the docs somewhere? Just checking).

Reply:
It's a tutorial on how to run a colocalization pipeline; what I wanted to make sure to include here is a link to more thorough explanations of what the different colocalization measurements MEAN, since many people don't know.  In general we haven't linked to our tutorials in the documentation for modules- presumably because keeping links up to date etc is a hassle.